### ISSUE TITLE
Asset Bundle Patterns not including SVG files?

### ISSUE THREAD

**Issue by cmosboss** [expo/expo#6050]
### Environment
  Expo CLI 3.1.2 environment info:
    System:
      OS: macOS 10.14.5
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 12.11.0 - ~/.nvm/versions/node/v12.11.0/bin/node
      Yarn: 1.6.0 - /usr/local/bin/yarn
      npm: 6.11.3 - ~/.nvm/versions/node/v12.11.0/bin/npm
    IDEs:
      Android Studio: 3.3 AI-182.5107.16.33.5264788
      Xcode: 11.1/11A1027 - /usr/bin/xcodebuild
    npmPackages:
      expo: ^34.0.1 => 34.0.4
      react: 16.8.3 => 16.8.3
      react-native: https://github.com/expo/react-native/archive/sdk-34.0.0.tar.gz => 0.59.8
      react-navigation: ^3.11.0 => 3.11.0
### Steps to Reproduce
Include 
```
    "assetBundlePatterns": [
      "**/*"
    ],
```
In your app.json
### Expected Behavior

Expect images / assets to be bundled including SVG's

### Actual Behavior

All of my assets (png, jpg etc) are included. However SVGs in the same directory are not. These SVGs are working via cloudfront CDN. However a large portion of my users have no internet access while using my app and it will not load SVGs as a result. Other images like PNG and JPG seem to load just fine.

I do not care particularly if my SVG's are optimized, I mostly want them bundled with the app so that they work offline.

**Comment by cruzach** [comment]
I just tested this with `expo-cli` v 3.3.0, and was able to bundle in SVGs to the binary (APK, didn't test IPA)

Can you make sure you're `Asset.loadAsync(require("./assets/images/[yoursvgfile].svg"))` your svg somewhere in your project?

**Comment by zocheyado** [comment]
I am not using Asset.loadAsync anywhere. Just simple requires. However adding Asset.loadAsync did not seem to make any difference for my SVG files. My assets are not located in the /assets directory though. They live under ./src/images, could that be my issue?

**Comment by cruzach** [comment]
`**/*` should match all directories besides `.` and `..` 
But in my test project, I put the SVG in the assets folder, so worth a shot

**Comment by cruzach** [comment]
@zocheyado were you able to figure out this issue?

**Comment by hyochan** [comment]
Having the same problem after migrating to `expo-assets`.

Below are the warning logs. Previously, it didn't appear.
```
Module "function SvgComponent(props) {
    return _react.default.createElement(_reactNativeSvg.default, (0, _extends2.default)({
      height: 48,
      viewBox: "0 0 48 48",
      width: 48
    }, props, {
      __source: {
        fileName: _jsxFileName,
        lineNumber: 4
      }
    }), _react.default.createElement(_reactNativeSvg.Path, {
      d: "M0 0h48v48H0z",
      fill: "none",
      __source: {
        fileName: _jsxFileName,
        lineNumber: 4
      }
    }), _react.default.createElement(_reactNativeSvg.Path, {
      d: "M26 14h-4v8h-8v4h8v8h4v-8h8v-4h-8v-8zM24 4C12.95 4 4 12.95 4 24s8.95 20 20 20 20-8.95 20-20S35.05 4 24 4zm0 36c-8.82 0-16-7.18-16-16S15.18 8 24 8s16 7.18 16 16-7.18 16-16 16z",
      __source: {
        fileName: _jsxFileName,
        lineNumber: 4
      }
    }));
  }" is missing from the asset registry
- node_modules/expo-asset/build/Asset.js:54:18 in fromModule
```

**Comment by cruzach** [comment]
@hyochan It's hard to debug without a reproducible demo. If you can share a snack that reproduces this error, that would help a lot

**Comment by cmosboss** [comment]
> @zocheyado were you able to figure out this issue?

I never did figure this out, I know it seems dependent on the operating system. I have not quite spent the time to figure this out. I know out of my current install base it happens only about ~1% of the time. Sorry I couldn't be more help.

**Comment by hyochan** [comment]
@cruzach We have a reproducible demo since we are working on opensource project together called [HackaTalk](https://github.com/dooboolab/hackatalk-mobile). 

I've managed to fix warning by removing `svgs` as below in [this commit](https://github.com/dooboolab/hackatalk-mobile/commit/16f471b4ccda21f7d25825a820a3f7fde92ef5fb).
![image](https://user-images.githubusercontent.com/27461460/73508864-a076bd80-4420-11ea-87b3-e6e83f32317f.png)

Previously, I didn't have to remove above.

**Comment by cruzach** [comment]
@hyochan are SVG files not being bundled into your binaries?

**Comment by cruzach** [comment]
@hyochan we'll need a minimal, reproducible demo in order to investigate, a full production project has too many variables at play that could cause issues. Here's a great guide on creating repros and why full projects don't really work- https://stackoverflow.com/help/minimal-reproducible-example

**Comment by cmosboss** [comment]
For what it's worth this issue changed to happening all of the time on all of my images when I migrated from SDK 33->37. My ultimate fix to that was an idea I got from your comment "Asset.loadAsync(require("./assets/images/[yoursvgfile].svg"))" I mentioned that I was NOT using that to load my images and I was just using simple requires. I ended up reworking all of my ~150 images to use Asset.loadAsync and it worked. But it still does not explain why the regular require worked for all images (png, jpg, etc) and even videos but SVG for whatever reason was left out in the cold.
